doc/static_discovery_configuration_guide.md

```diff
@@ -0,0 +1,808 @@
+sudo keytool -import -alias org1_gw -file org1_tls_certificate.cer -keystore /etc/harmony-ap/tls-truststore.jks -storepass <tls_truststore_password>
+```
+
+Set the file permissions of the new TLS truststore file:
+
+```
+sudo chown harmony-ap:harmony-ap /etc/harmony-ap/tls-truststore.jks
+sudo chmod 751 /etc/harmony-ap/tls-truststore.jks
+```
+
+### 3.7 Configure One-Way SSL
+
+This step requires shell access to the host.
+    
+On Access Points 1 (`org1_gw`) and Access Point 2 (`org2_gw`), create the `/etc/harmony-ap/clientauthentication.xml` 
+file with the following content:
+
+```
+<http-conf:tlsClientParameters disableCNCheck="true" secureSocketProtocol="TLSv1.2"
+                               xmlns:http-conf="http://cxf.apache.org/transports/http/configuration"
+                               xmlns:security="http://cxf.apache.org/configuration/security">
+    <security:trustManagers>
+        <security:keyStore type="JKS" password="<tls_truststore_password>"
+                           file="/etc/harmony-ap/tls-truststore.jks"/>
+    </security:trustManagers>
+</http-conf:tlsClientParameters>
+```
+
+**Note:** Use the password (`tls_truststore_password`) created in step 3.6.
+
+Set the file permissions of the new `/etc/harmony-ap/clientauthentication.xml` file:
+
+```
+sudo chown harmony-ap:harmony-ap /etc/harmony-ap/clientauthentication.xml
+sudo chmod 751 /etc/harmony-ap/clientauthentication.xml
+```
+
+### 3.8 Apply Changes
+
+This step requires shell access to the host.
+
+On Access Points 1 (`org1_gw`) and Access Point 2 (`org2_gw`), restart the `harmony-ap` service to apply the 
+configuration changes:
+
+```
+sudo systemctl restart harmony-ap
+```
+
+### 3.9 Send Test Message
+
+Send a request to Access Point 1 (`org1_gw`) using the curl command below. The request (`submitRequest.xml`) can be downloaded [here](configuration_examples/static_discovery/submitRequest.xml). The content inside the payload's `value` element must be [base64 encoded](https://www.base64encode.org/).
+
+```
+curl -u org1:<org1_plugin_user_password> --header "Content-Type: text/xml;charset=UTF-8" --data @submitRequest.xml https://<AP1_IP_OR_FQDN>:8443/services/backend -v -k
+```
+
+A successful response looks like this:
+
+```
+<?xml version="1.0" encoding="UTF-8"?>
+<soap:Envelope xmlns:soap="http://www.w3.org/2003/05/soap-envelope">
+   <soap:Body>
+      <ns2:submitResponse xmlns:ns2="http://org.ecodex.backend/1_1/" xmlns:ns3="http://docs.oasis-open.org/ebxml-msg/ebms/v3.0/ns/core/200704/" xmlns:ns4="http://www.w3.org/2003/05/soap-envelope" xmlns:xmime="http://www.w3.org/2005/05/xmlmime">
+         <messageID>bd3ea132-6a0d-11ec-9789-0af9f1d3371a@edelivery.digital</messageID>
+      </ns2:submitResponse>
+   </soap:Body>
+</soap:Envelope>
+```
+
+List received and pending messages on the Access Point 2 (`org2_gw`). The request (`listPendingMessagesRequest.xml`) can be downloaded [here](configuration_examples/static_discovery/listPendingMessagesRequest.xml).
```

Comment:
It might be better to link to the raw file in case someone just wants to grab the link and run wget or curl with it. Currently, it would give the github HTML page.

Reply:
I updated the link to point to the raw file.